### PR TITLE
Python Lab: Start mode and eyes tests

### PR DESCRIPTION
**Pull request by molly-moen**
More Python Lab tests:
- UI test for start mode
- Eyes test for start mode
- Eyes test for basic run

While working on this I noticed the extra links button was using a legacy button component, I think this was an oversight on my part. I converted it to use the component library button component instead, so it will not be purple instead of orange. I updated the styling of the new version of the button so it still sits above the copyright footer.

New extra links button:
Screenshot 2024-10-24 at 3 15 54 PM

## Links

- jira ticket: [CT-726](https://codedotorg.atlassian.net/browse/CT-726), [CT-725](https://codedotorg.atlassian.net/browse/CT-725)

## Testing story
Tested via saucelabs tunnel

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by ebeastlake — APPROVED** [review]

File: dashboard/test/ui/features/code_tools/pythonlab/pythonlab_run_eyes.feature [hunk 1]
```diff
@@ -0,0 +1,18 @@
+@no_mobile
+# Our minimum version of Safari does not support web workers
+@no_safari
+@eyes
```
Comment (breville) [inline dashboard/test/ui/features/code_tools/pythonlab/pythonlab_run_eyes.feature:4]: Out of curiosity, would an eyes test run on Safari anyway?

**Review by breville — APPROVED** [review]
Great to have this testing in place.

File: dashboard/test/ui/features/code_tools/pythonlab/pythonlab_run_eyes.feature (hunk 1 above)
Comment (molly-moen) [reply dashboard/test/ui/features/code_tools/pythonlab/pythonlab_run_eyes.feature:4]: Yeah I wasn't sure if this was needed...I copied from the ui tests, I can remove this

Comment (molly-moen) [reply dashboard/test/ui/features/code_tools/pythonlab/pythonlab_run_eyes.feature:4]: I think I'll leave it, when you run locally it'll try to run against Safari so this makes it explicit.

**Review by thomasoniii — APPROVED** [review]
lgtm 🚀

**Pull request merged**